sidebars.js

```diff
@@ -135,7 +135,8 @@ module.exports = {
         'develop/tools-and-features/feeds',
         'develop/tools-and-features/pss',
         'develop/tools-and-features/bee-dev-mode',
-        'develop/tools-and-features/starting-a-test-network'
+        'develop/tools-and-features/starting-a-test-network',
+        'develop/tools-and-features/act'
```

Comment:
'act' as an Id is not colliding with the Overview page? Did you try in the developer mode if the link appears at the right place?

Reply:
I tried it out, it works